src/main/java/com/palantir/gradle/versions/FixLegacyJavaConfigurationsPlugin.java

```diff
@@ -88,26 +77,28 @@ private void injectVersions(Configuration conf, GetVersion getVersion) {
             }
             // Code adapted from:
             // https://github.com/nebula-plugins/nebula-dependency-recommender-plugin/blob/64ed7c6853f80b909918e6a595231a5e9803ae8b/src/main/groovy/netflix/nebula/dependency/recommender/DependencyRecommendationsPlugin.java
-            conf.getResolutionStrategy().eachDependency(details -> {
-                ModuleVersionSelector requested = details.getTarget();
+            conf.getResolutionStrategy()
+                    .eachDependency(details -> {
```

Comment:
Do you think this is fixable to move up to the previous line @dansanduleac?

Reply:
yep